### PR TITLE
Update Gradle version to 8

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -47,9 +47,9 @@ dependencies {
     implementation(libs.logback.classic)
 }
 
-kotlin {
-    jvmToolchain {
-        check(this is JavaToolchainSpec)
-        languageVersion.set(JavaLanguageVersion.of(8))
-    }
-}
+//kotlin {
```
Comment (rsinukov) [inline buildSrc/build.gradle.kts:50]: why commented?

**Review by rsinukov — CHANGES_REQUESTED** [review]
Please check CI

File: buildSrc/build.gradle.kts (hunk 1 above)
Comment (e5l) [reply buildSrc/build.gradle.kts]: fixed

**Comment by rsinukov** [comment]
Please rebase

**Comment by e5l** [comment]
done

**Review by rsinukov — APPROVED** [review]
LGTM, but please wait for the CI to run

File: buildSrc/build.gradle.kts [hunk 2]
```diff
@@ -3,8 +3,8 @@
 */
 
 plugins {
-    id("org.gradle.kotlin.kotlin-dsl") version "3.2.4"
-    kotlin("plugin.serialization") version "1.7.21"
+    kotlin("plugin.serialization") version "1.8.20"
+    id("org.gradle.kotlin.kotlin-dsl") version "4.0.14"
```
Comment (hfhbd) [inline buildSrc/build.gradle.kts:7]: you should not set this version explicitly, but use `kotlin-dsl`. the same for `plugin-serialization` should use `embeddedKotlinVersion`.

**Comment by hfhbd** [comment]
Did you also run the task twice? The wrapper wasn't update too. Wanna have a PR?

**Comment by e5l** [comment]
Strange, will take a look

**Pull request merged**